docs/output.md

```diff
@@ -11,6 +11,13 @@ This document describes the output produced by the pipeline. Most of the plots a
 The directories listed below will be created in the results directory after the pipeline has finished. All paths are relative to the top-level results directory.
 
 <!-- TODO nf-core: Write this documentation describing your workflow's output -->
+Reading this you likely have already experienced first hand that single-cell sequencing data is big, exceeding the amount of RAM available to most computers. Nonetheless, algorithms may demand expression values of a particular set of genes for a particular set of cells at any time.
+The [hiearchical data format 5](https://en.wikipedia.org/wiki/Hierarchical_Data_Format) (hdf5) thus provides the data in a structured manner with indices such that computational demands are mnimized to retrieve individual records. And much like a traditional file system, or like an excel sheet with many named tabs, different pieces of information can be integrated into a single consistent ..AnnData object that is effectively mapped to a file on the hard drive.
```

Comment:
```suggestion
The [hiearchical data format 5](https://en.wikipedia.org/wiki/Hierarchical_Data_Format) (hdf5) thus provides the data in a structured manner with indices such that computational demands are mnimized to retrieve individual records. And much like a traditional file system, or like an excel sheet with many named tabs, different pieces of information can be integrated into a single consistent ..AnnData object that is effectively mapped to a file on the hard drive.
```
The [hierarchical data format 5 (hdf5)](https://en.wikipedia.org/wiki/Hierarchical_Data_Format) provides a simple interface for storing combinations of sparse and dense matrices. A special implementation of the `hdf5` format is the AnnData (`h5ad`) format, which was specifically designed for storing single-cell (transcriptomics) data. It can hold multiple layers of sparse expression data, as well as additional metadata for the observations (cells) and variables (genes). Detailed information can be found in the [AnnData documentation](https://anndata.readthedocs.io/en/latest/), but here are the most important parts:
- `.obs`: Dataframe with metadata for cells
- `.var`: Dataframe with metadata for genes
- `.obsm`: Can hold multiple named numpy arrays. These represent low-dimensional embeddings of the cells (PCA, UMAP, integrations, etc.)
- `.varm`: Same as `.obsm`, just for genes